code-analysis/pom.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
```

Comment:
This setup is recommended and necessary afaik for multimodule projets, see https://maven.apache.org/plugins/maven-pmd-plugin/examples/multi-module-config.html for details